src/test/script_tests.cpp

```diff
@@ -306,10 +306,10 @@ class TestBuilder
         return *this;
     }
 
-    TestBuilder& Add(const CScript& _script)
+    TestBuilder& Add(const opcodetype& _op)
```

Comment:
Maybe this method should be called Opcode() instead of Add() to be consistent with the Num() method below and since it's no longer doing + operation

Reply:
Will do when I rebase

Reply:
Renamed to `Opcode`